packages/apollo-server-core/src/ApolloServer.ts

```diff
@@ -261,6 +261,21 @@ export class ApolloServerBase<
         : noIntro;
     }
 
+    if (
```

Comment:
move to right before setting unboundedcache

Reply:
not that it really matters but my idea was it could be nested inside the `if (!requestOptions.cache)` below instead of repeating that part of the condition